test/test_nestedtensor.py

```diff
@@ -0,0 +1,146 @@
+import torch
+import unittest
+from common_utils import TEST_WITH_PROTOTYPES
+
+if TEST_WITH_PROTOTYPES:
```

Comment:
Might be useful to set up a CI job next

Reply:
Yes, but I'd maybe do that after it's become its own thing.